Shared_Funcs/pemfc_pre.py

```diff
@@ -52,15 +52,15 @@
 # Change parameters for optimization only:
 if optimize == 1:
     if tog == 2:
-        R_naf = c*w_Pt**d
+        R_naf = (c*w_Pt**d + 35) *1e-3
```

Comment:
What does the number 35 refer to, here?  Generally, I don't like to see hard-coded numbers in the code, unless there is a very good reason.  Otherwise it should be set as a variable somewhere more visible to the user, possibly even as a user input (even if it is one that the user should never manipulate).

Reply:
The 35 was from the by-hand fit that I did. I assumed that the membrane had to have some resistance even near zero Pt loading. To be as comparable to the by-hand fit as possible I went ahead and left that in there. Do you think it makes more sense to take it out and still just have the c and d parameters without that shift?

Reply:
I think there are two suitable approaches, here:

1. Add it as an additional fitting parameter in the routine; or
2. Create it as an input parameter that is set somewhere else, but not fit.

Removing the shift entirely seems like it would produce qualitatively different behavior.  Adding it as a fitting parameter, obviously, would allow the fitting to determine if eliminating it was a better approach.  

Assuming it doesn't slow the fitting down by too much, I guess I'd opt for option 1.

Reply:
I don't think that it will slow it down by much. I'll update it and rerun for that case and push another commit when it finishes.